internal/entrypoint/run.go

```diff
@@ -154,23 +158,27 @@ func Run(ctx context.Context, config *Config, exporter otlexporters.Otlexporter,
 				logger.Info("powerflex volume metrics collection is disabled")
 				continue
 			}
-			_, err := config.PowerFlexClient.Authenticate(&config.PowerFlexConfig)
-			if err != nil {
-				logger.WithError(err).Error("failed to authenticate with PowerFlex. retrying on next tick...", err)
-				continue
-			}
-			sdcs, err := pflexSvc.GetSDCs(ctx, config.PowerFlexClient, config.SDCFinder)
-			if err != nil {
-				logger.WithError(err).Error("getting SDCs")
-				continue
-			}
 
-			volumes, err := pflexSvc.GetVolumes(ctx, sdcs)
-			if err != nil {
-				logger.WithError(err).Error("getting volumes")
-				continue
+			for key, client := range config.PowerFlexClient {
+				sioConfig := config.PowerFlexConfig[key]
+				_, err := client.Authenticate(&sioConfig)
+				if err != nil {
+					logger.WithError(err).Error("failed to authenticate with PowerFlex. retrying on next tick...")
```

Comment:
For debugging any authentication errors for multi-array, it would be helpful to log the endpoint of the PowerFlex as a field in this log message.